wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,440 @@
+
+See [Kandashi's Active Effects Guide](https://docs.google.com/document/d/1DuZaIFVq0YulDOvpahrfhZ6dK7LuclIRlGOtT0BIYEo) for a trove of useful information
+
+This document only covers Active Effects available to the Core dnd5e System.
+
+## Legend
+
+`[number]` - These square brakets mean "replace this with your value of the type within the brackets".
+So this example: `+[number]` would mean you input `+3`.
+
+`[formula]` - When `formula` is mentioned in this document it means this value can be populated with any dice formula. For example, Bless adds several effects with the Effect Value of `1d4`.
+
+
+As part of this, an Actor's Rolldata is available as ["@attributes."](https://github.com/foundryvtt/dnd5e/wiki/Roll-Formulas) Useful examples:
+
+| @attribute                 | Description                   |
+| -------------------------- | ----------------------------- |
+| `@abilities.dex.mod`       | Actor's Dexterity Modifier    |
+| `@prof`                    | Actor's Proficiency Bonus     |
+| `@details.level`           | Actor's overall Level         |
+| `@classes.barbarian.levels` | Actor's Barbarian Class Level |
+
+  
+> Note that when using formulas in an Active Effect Value, the actor sheet display that corresponds to the changed value will not display the evaluated formula, but it will be applied when rolled.
+> E.g. When adding `+@abilities.cha.mod` to `system.bonuses.abilities.save` to simulate a Paladin's Aura of Protection, the actor sheet will not display that bonus applied to saving throws. The bonus will be present when the saving throw is rolled.
+
+To find out more about these, this post from Unsoluble in the [FoundryVTT Discord server](https://discord.gg/foundryvtt)'s #core-how-to channel can help:
+
+> To explore the data model in order to refer to inline properties like character levels, attributes, and other values, here are a few approaches:
+>
+> • Select a token, then open up the dev tools (F12 on Win; ⌥⌘I on Mac), and paste this into the Console:
+`console.log(canvas.tokens.controlled[0].actor.getRollData());`
+>
+> • Or: Install the "Autocomplete Inline Properties" module, to be able to just start typing in a supported field and have the available properties pop up (not all systems supported yet).
+>
+> • Or: Install this module (by pasting this manifest URL into the bottom of the Install Module window), enable it in your world, find and import its macro from the bundled Compendium, and run it on the selected token. `https://raw.githubusercontent.com/relick/FoundryVTT-Actor-Attribute-Lists/master/module.json`
+>
+> • Or: Right-click an actor in the sidebar and choose Export Data, which will get you a JSON file you can browse through. 
+
+
+## Commonly Desired Effect Examples
+
+### Abilities
+
+```
+system.abilities.[abbreviation].value
+                                bonuses.check
+                                        save
+```
+
+
+> <details>
```

Comment:
Does this need to be inside a `<blockquote>`? Seems like it would look fine outside it too. Don't feel strongly though.

Reply:
This also comes from Calego's original doc, where it was in a `:::info` block to have some unique styling, which renders as such on hack.md
![image](https://github.com/foundryvtt/dnd5e/assets/86370342/06e27d1b-2660-47f8-9784-064ae59aef88)

I agree that it isn't really doing much for us here, but for some reason I do like it...

Reply:
If you prefer it, then by all means keep it